MaxMind.MinFraud/Request/CreditCard.cs

```diff
@@ -49,23 +53,6 @@ public sealed class CreditCard
             string? token = null
         )
         {
-            if (issuerIdNumber != null && !IssuerIdNumberRe.IsMatch(issuerIdNumber))
```

Comment:
I'm guessing this validation will still run if we create the object in the old way too?

Reply:
Yep, all of the constructors initialize through the properties.